app/lib/govuk_elements_form_builder/form_builder.rb

```diff
@@ -5,13 +5,14 @@ class FormBuilder < ActionView::Helpers::FormBuilder # rubocop:disable Metrics/C
     # https://guides.rubyonrails.org/configuring.html#configuring-action-view
     ActionView::Base.field_error_proc = proc { |html_tag| html_tag.html_safe }
 
-    CUSTOM_OPTIONS = %i[label input_prefix field_with_error hint title inline].freeze
+    CUSTOM_OPTIONS = %i[label input_prefix field_with_error hint title inline legacy_method].freeze
```

Comment:
I wonder if `text_input` would be a better label. I'm not sure what these methods are a legacy of.

Reply:
Yes that's better